source/index-warning.txt

```diff
@@ -103,14 +98,28 @@ Explorer Playgrounds` screen. For example:
 Disable Index Warning
 ~~~~~~~~~~~~~~~~~~~~~
 
-To disable the index warning in the plugin:
+To disable the index warning in the side panel:
```

Comment:
(info typing)
Again, these should probably be info typed. Happy to chat if you want.

Reply:
Moved to its own task page!